qemu/tests/hotplug_mem_repeat.py

```diff
@@ -45,11 +45,18 @@ def start_test(self):
             arg = "movable_node"
             utils_test.update_boot_option(vm, args_added=arg)
         original_mem = self.get_guest_total_mem(vm)
+
+        rhel_version = "\"Red Hat Enterprise Linux Server release 7\""
+        cmd = "grep %s /etc/redhat-release" % rhel_version
+        status, output = session.cmd_status_output(cmd)
```

Comment:
Suggest to move these lines into `if` block in L44.

Reply:
Updated